data-labeling/environments/data-labeling/context/Dockerfile

```
@@ -0,0 +1,25 @@
+FROM mcr.microsoft.com/azureml/openmpi4.1.0-ubuntu20.04:{{latest-image-tag}}
+
+WORKDIR /
+
+ENV CONDA_PREFIX=/azureml-envs/labeling-base-ubuntu20.04-py39-cpu
```

Consider a simpler directory name:
```suggestion
ENV CONDA_PREFIX=/azureml-envs/labeling-base
```